doc/HttpLuaModule.wiki

```diff
@@ -5805,6 +5805,26 @@ This function returns a Lua number for the process ID (PID) of the current Nginx
 
 This API was first introduced in the <code>0.9.5</code> release.
 
+== ngx.worker.count ==
+
+'''syntax:''' ''count = ngx.worker.count''
+
+'''context:''' ''set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, body_filter_by_lua*, log_by_lua*, ngx.timer.*, init_by_lua*''
+
+Returns the count number of Nginx worker processes.
+
+This API was first introduced in the <code>0.9.17</code> release.
```

Comment:
Hmm, this patch has already missed the 0.9.17 release's merge window. Hopefully it will get included in the next release. No need to update the version number on your side. I can change this to the right number while merging myself :)